cime/scripts/lib/CIME/provenance.py

```diff
@@ -17,12 +17,14 @@ def _get_batch_job_id_for_syslog(case):
     """
     mach = case.get_value("MACH")
     try:
-        if mach in ['anvil', 'titan']:
+        if mach in ['titan']:
```

Comment:
Should you just remove this line since Titan is no more?

Reply:
Done.